join_github_app/main/services/github_service.py

```diff
@@ -20,8 +20,11 @@ def __init__(self, org_token: str) -> None:
         )
 
     def send_invites_to_user_email(self, email: str, organisations: list) -> None:
+        valid_orgs = [ organisation.name for organisation in app_config.github.organisations if organisation.enabled ]
         for organisation in organisations:
-            if app_config.github.send_email_invites_is_enabled:
+            if organisation in valid_orgs and app_config.github.send_email_invites_is_enabled:
                 self.github_client_core_api.get_organization(organisation.lower()).invite_user(email=email)
-            else:
+            elif not app_config.github.send_email_invites_is_enabled:
                 logger.info("Not sending invitation for organisation [ %s ] as SEND_EMAIL_INVITES is [ %s ]", organisation, app_config.github.send_email_invites_is_enabled)
+            else:
+                logger.info("Not sending invitation for organisation [ %s ] as selected organisation is invalid", organisation)
```

Comment:
```suggestion
                logger.info("Invitation for organisation [ %s ] not sent as selected organisation is invalid", organisation)
```
I've changed the wording here in my PR; you might need to update with main to pull the other change in